### PR TITLE
Feat: static pages

### PR DESCRIPTION
**Pull request by yashpatel1797**
All pages are static
- Feed page
- Explore page
- Header
- Sidebar
- PostCard
- Notification page
- Bookmarks page
- Profile page
- Login/signup page

### CODE REVIEW

File: src/components/Navbar/BottomNav.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```
Comment (devdeadviz) [inline src/components/Navbar/BottomNav.js:1]: You don't need to import React in every file, it's available automatically.

**Review by sashaboi — APPROVED** [review]
Looks amazing ! 

A few suggestions : 
1.Background color seems to interfere with selected buttons, 
2.add the redirects file for netlify redirects

**Review by sharmayatendra — COMMENTED** [review]
code looks clean
file structure is good
design looks consistent

File: src/components/PostCard/PostCard.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        <span className="material-icons">
+                            favorite_border
+                        </span>
+                    </button>
+                </div>
+                <div>
+                    <button className='w-10 h-10 flex justify-center items-center hover:text-sky-50 hover:bg-sky-500/25 rounded-full'>
+                        <span className="material-icons">
+                            chat_bubble_outline
+                        </span>
```
Comment (devdeadviz) [inline src/components/PostCard/PostCard.js:50]: Use code formatter.

File: src/components/ProfileCard/ProfileCard.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    <div className='mt-3'>
+                        <p>status</p>
+                        <p>more details</p>
+                        <p>more details</p>
+                    </div>
+                </div>
+
+
+            </div>
+            {/* </div> */}
```
Comment (devdeadviz) [inline src/components/ProfileCard/ProfileCard.js:52]: Remove commented code

**Review by devdeadviz — APPROVED** [review]
The code is clean, use any code formatter to maintain the consistent format throughout your all files.

**Comment by devdeadviz** [comment]
Capture
There is some style issue with your sidebar, you can see your selected tab text is not visible.

**Pull request merged**